engine/evarutil.ml

```diff
@@ -193,7 +161,9 @@ let is_ground_env = memo is_ground_env
 
 exception NoHeadEvar
 
-let head_evar =
+let head_evar sigma c =
```

Comment:
Probably ought to be deprecated. It is only used twice in Evarsolve in a fishy way (I remember it breaking at some point in the econstr transition), and twice in Evar_tactics, which is already deprecated. Maybe the good moment to seal the API away.